plugins/backend/catalog-backend-module-aws/README.md

```diff
@@ -0,0 +1,4 @@
+# Catalog Backend Module for AWS
+
+This is an extension module to the plugin-catalog-backend plugin, providing
+processors to read AWS objects as Backstage Entities
```

Comment:
Could you pls explain here how can they setup the providers to be able to use these ? Pretty much what you did in the catalog.ts.
Is there an opportunity to abstract that wiring logic away and expose some kind of helper function to help setting it up ?

Reply:
done.